assets/js/src/core/modules/element/editor/tab-manager/interface/IEditorTab.ts

```diff
@@ -12,11 +12,15 @@
 */
 
 import type React from 'react'
+import { type Permissions } from '@Pimcore/modules/asset/asset-api-slice.gen'
```

Comment:
Same as above - here we are in the element namespace but only support the asset permissions.